taiga_contrib_ldap_auth_ext/connector.py

```diff
@@ -30,7 +31,7 @@ class LDAPUserLoginError(LDAPError):
 
 
 # TODO https://github.com/Monogramm/taiga-contrib-ldap-auth-ext/issues/16
-SERVER = getattr(settings, "LDAP_SERVER", "localhost")
+SERVERS = getattr(settings, "LDAP_SERVERS", "localhost")
```

Comment:
@AminovE99 Instead of replacing `SERVER` by `SERVERS` and thus breaking setup of all Taiga environments which do not update their config, you should offer an automatic conversion and be compatible with initial setup.